<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="TRD-LAP-80" failures="1" tests="5" name="Test" time="0.626" errors="0" timestamp="2024-07-01T11:37:45 IST">
  <testcase classname="Testng_package.Testng_demo" name="demo" time="0.036">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Testng_package.Testng_demo.demo(Testng_demo.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
  <testcase classname="Testng_package.Testng_2" name="day" time="0.019"/>
  <testcase classname="Testng_package.Testng_4" name="Webloginhome" time="0.011"/>
  <testcase classname="Testng_package.Testng_4" name="loginapihome" time="0.013"/>
  <testcase classname="Testng_package.Testng_4" name="mobloginhome" time="0.002"/>
  <testcase classname="Testng_package.Testng_demo" name="seconddemo" time="0.0">
    <ignored/>
  </testcase> <!-- seconddemo -->
</testsuite> <!-- Test -->
